CRM/Core/Config/MagicMerge.php

```diff
@@ -232,9 +232,11 @@ public function __get($k) {
 
       case 'setting-url-abs':
       case 'setting-url-rel':
+      case 'setting-url-rel-noslash':
```

Comment:
I agree this is totally consistent with the style/precedent.

The current test failure is arises because it's coded to diff type-names (`setting-url-rel-noslash` vs `setting-url-abs-noslash`).

Looking at this code again (with some distance), it's not actually necessary to add items for each permutation. In the `setting-path` examples, we pass some optional arguments (`mkdir`, `restrict`). We could just have type `setting-url` with optional arguments (`rel`, `noslash`).

https://github.com/lcdservices/civicrm-core/pull/1